Marlin/pins_GT2560.h

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
I think that the pinout it not the same. Must double check, but if it was my printer should have worked the first time. I can send you the schematic if you want to check. Thanks !